variants/heltec_wireless_tracker/variant.h

```diff
@@ -0,0 +1,53 @@
+#define LED_PIN 18
+
+#define TFT_POWER_EN 46
+
+#define ST7735_RESET 39 // Output
+#define ST7735_CS 38
+#define ST7735_BACKLIGHT_EN 45
+#define ST7735_RS 40
+#define ST7735_SDA 42
+#define ST7735_SCK 41
+
+// #define RESET_OLED 21
+// #define I2C_SDA 17 // I2C pins for this board
+// #define I2C_SCL 18
+
+#define SCREEN_TRANSITION_FRAMERATE 1 // fps
+
+#define VEXT_ENABLE Vext // active low, powers the oled display and the lora antenna boost
+#define BUTTON_PIN 0
+
+#define BATTERY_PIN 1 // A battery voltage measurement pin, voltage divider connected here to measure battery voltage
+#define ADC_CHANNEL ADC1_GPIO1_CHANNEL
+#define ADC_ATTENUATION ADC_ATTEN_DB_2_5 // lower dB for high resistance voltage divider
+#define ADC_MULTIPLIER 4.9
+
+#undef GPS_RX_PIN
+#undef GPS_TX_PIN
+#define GPS_RX_PIN 33
```

Comment:
Are these RX/TX pins correct? In the heltec-tracker pinout it is TX=33, RX=34

Reply:
According to this pins_arduino.h - https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/blob/master/esp32/variants/wireless_paper/pins_arduino.h, you could leave them under and test

Reply:
This would be the proper pins_arduino.h for the tracker;
https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/blob/master/esp32/variants/wireless_track/pins_arduino.h

And there is and TX=33 RX=34 definition.

Reply:
#define GPS_RX_PIN 33
#define GPS_TX_PIN 34
 is correct